R/weightedMeanByID.R

```diff
@@ -0,0 +1,86 @@
+#' @title Aggregate different rows assigned to the same ID by calculating a 
+#' weighted mean 
+#'
+#' @description After calculating row means to summarize across replicates, 
```

Comment:
Maybe change replicates to 'samples' (as it's averaging across all samples, without taking a grouping variable into account)